{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -11,13 +11,13 @@ def root_factory(request):
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
-    config = Configurator(root_factory=root_factory, settings=settings)
-    settings = config.get_settings()
-    settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
-    config.include('pyramid_chameleon')
-    config.include('pyramid_tm')
-    config.include('pyramid_retry')
-    config.include('pyramid_zodbconn')
-    config.add_static_view('static', 'static', cache_max_age=3600)
-    config.scan()
-    return config.make_wsgi_app()
+    with Configurator(root_factory=root_factory, settings=settings) as config:
+        settings = config.get_settings()
+        settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
```

Comment:
1) Move the settings above the config so you don't have to get them then set them.
2) Use `config.set_root_factory` instead of `root_factory=`.

```python
def main(global_config, **settings):
    settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager']
    with Configurator(settings=settings) as config:
        config.set_root_factory(...)
        ...
```